src/ziggurat/streams.clj

```diff
@@ -2,29 +2,26 @@
   (:require [clojure.tools.logging :as log]
             [mount.core :as mount :refer [defstate]]
             [ziggurat.channel :as chl]
-            [ziggurat.config :refer [ziggurat-config]]
+            [ziggurat.config :refer [get-in-config ziggurat-config build-all-config-properties build-streams-config-properties]]
```

Comment:
we can remove `build-all-config-properties` from here.

Reply:
done